pkg/agent/proxy/metrics/metrics.go

```diff
@@ -0,0 +1,117 @@
+package metrics
+
+import (
+	"sync"
+
+	kmetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog"
+)
+
+var (
+	once sync.Once
+
+	subsystemProxy = "antrea_proxy"
+
+	SyncProxyLatency = kmetrics.NewHistogram(
+		&kmetrics.HistogramOpts{
+			Subsystem:   subsystemProxy,
```

Comment:
Using subsystem is a good idea. 
Why not use namespace as "antrea"? Do you see any issue?

Reply:
Good to know this. I didn't realize this field before. Maybe we can have another PR to unify them all?

Reply:
I think that we should use namespace and subsystem in all the metrics, in a separate PR.

Reply:
@weiqiangt A different PR to make all metrics follow this convention makes sense.

Reply:
we can do that, but is it considered a breaking change?

Reply:
I think the metric name user faced will not be changed, but the metric name will not contains `antrea`/`agent`/`controller`.